src/main/java/org/mule/module/apikit/validation/attributes/HeadersValidator.java

```diff
@@ -233,7 +233,7 @@ public static boolean someRepresentationIsSupported(List<String> supportedRepres
   }
 
   private static boolean isCompatible(MimeType expected, MimeType supported) {
-    return isCompatible(expected.getType(), supported.getType()) && isCompatible(expected.getType(), supported.getType());
+    return isCompatible(expected.getType(), supported.getType()) && isCompatible(expected.getSubtype(), supported.getSubtype());
```

Comment:
There's a slight bug here still present. `isCompatible("*/png", "image/png")` doesn't really make sense. Should we return false in that case?